src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1119,6 +1119,12 @@ private String execDockerCmd(String command, int timeout) {
             if (allLines.length() > 0) {
                 result = allLines.toString();
             }
+        } catch (IllegalThreadStateException  e) {
+            // the timeout was too short and the docker command has not yet completed. There is no exit value.
+            debug("IllegalThreadStateException, message="+e.getMessage());
+            error("The docker command did not complete within the timeout period: " + timeout + " seconds. " +
+                "Use the dockerTimeout option to specify a longer period or investigate issues running docker. ", e);
```

Comment:
"investigate issues running docker" sounds kind of vague.  If possible, be more specific on what the user should do.  Although this `execDockerCmd` method is used for docker build, docker ps, and docker stop so the user action may not be the same for all cases.

Or, perhaps just "investigate issues with running the docker command." if we need to be generic here.

Reply:
Check out the new message.